src/ukify/test/test_ukify.py

```diff
@@ -521,12 +521,89 @@ def test_addon(tmp_path):
 
     assert found is True
 
+
+def test_sign_uki(kernel_initrd, tmp_path, signing_crt_key):
+    if kernel_initrd is None:
+        pytest.skip('linux+initrd not found')
+
+    # Build a UKI first
+    first_uki_output = f'{tmp_path}/basic.efi'
+    opts = ukify.parse_args([
+        'build',
+        *kernel_initrd,
+        f'--output={first_uki_output}',
+        '--uname=1.2.3',
+        '--cmdline=ARG1 ARG2 ARG3',
+        '--os-release=K1=V1\nK2=V2\n',
+        '--section=.test:CONTENTZ',
+    ])
+
+    try:
+        ukify.check_inputs(opts)
+    except OSError as e:
+        pytest.skip(str(e))
+
+    ukify.make_uki(opts)
+
+    cert, key = signing_crt_key
+
+    output = f'{tmp_path}/signed.efi'
+    opts = ukify.parse_args([
+        'sign',
+        f'--uki={first_uki_output}',
+        f'--output={output}',
+        '--cmdline=ARG1 ARG2 ARG3',
+        '--append-cmdline',
+        f'--secureboot-certificate={cert.name}',
+        f'--secureboot-private-key={key.name}',
+        '--sign-kernel'
+    ])
+    ukify.sign_uki(opts)
+
+    # let's check that objdump likes the resulting file
+    dump = subprocess.check_output(['objdump', '-h', output], text=True)
+
+    for sect in 'text cmdline test sbat'.split():
```

Comment:
Let's not write perl. :)
```suggestion
    for sect in ['text', 'cmdline', 'test', 'sbat']:
```